catalog/imaging/imaging-plugin-nitf/pom.xml

```diff
@@ -110,7 +110,7 @@
         <dependency>
             <groupId>ddf.lib</groupId>
             <artifactId>spock-shaded</artifactId>
-            <version>${ddf.version}</version>
+            <version>2.11.6</version>
```

Comment:
What was this switched to a release version?

Reply:
Weird, don't remember doing that. I'll fix it really quick